org.sonarlint.eclipse.core.tests/src/test/java/org/sonarlint/eclipse/core/internal/jobs/AnalyzeStandaloneProjectJobTest.java

```diff
@@ -111,7 +111,10 @@ public void analyzeWithRuleParameters() throws Exception {
     FileWithDocument fileToAnalyze = new FileWithDocument(new DefaultSonarLintFileAdapter(slProject, file), document);
     RuleConfig ruleConfig = new RuleConfig("javascript:S100", true);
     ruleConfig.getParams().put("format", "^[0-9]+$");
-    SonarLintGlobalConfiguration.saveRulesConfig(asList(ruleConfig));
+    SonarLintGlobalConfiguration.saveRulesConfig(asList(
+        ruleConfig,
+        // disable multi-line issue because SimpleDocument does not implement getLineOffset()
```

Comment:
Since you have implemented getLineOffset, maybe now you can keep the rule `javascript:S1186`?

Reply:
Good idea :+1: